src/tests/ftest/object/object_integrity.py

```diff
@@ -133,6 +134,65 @@ def reconnect(self):
                                self.obj,
                                objtype=4)
 
+    @avocado.fail_on(DaosApiError)
+    def test_invalid_tx_commit_close(self):
+        """
+        Test ID:
+            (1)DAOS-1346: Verify commit tx bad parameter behavior.
+            (2)DAOS-1343: Verify tx_close bad parameter behavior.
+            (3)DAOS-1342: Verify tx_close through daos_api.
+        Test Description:
+            Write Avocado Test to verify commit tx and close tx
+                          bad parameter behavior.
+        :avocado: tags=negative_test,neg_tx_commit,medium,vm
```

Comment:
I think we can include tags all, pr,small to this. Also that will make sure this test is run in CI CI before landing.

Reply:
Have updated script.